Packs/Slack/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Slack",
     "description": "Interact with Slack API - collect logs, send messages and notifications to your Slack team.",
     "support": "xsoar",
-    "currentVersion": "3.4.19",
+    "currentVersion": "3.4.20",
```

Comment:
let's bump a minor, this change is more significant than a revision.

Reply:
I have bumped up the minor version from `3.4.19` to `3.5.0` in commit 86a9eb17e04ab27552e682f68301ea4abf685423 using the demisto-sdk